protocol-designer/src/assets/localization/en/tooltip.json

```
@@ -55,7 +55,8 @@
       "preWetTip": "Pre-wet pipette tip by aspirating and dispensing 2/3 of the tip's max volume",
       "volume": "Volume to dispense in each well",
       "blowout_z_offset": "The height at which blowout occurs from the top of the well",
-      "blowout_flowRate": "Blowout speed"
+      "blowout_flowRate": "Blowout speed",
+      "setTemperature": "Select the temperature to set your module to"
```

let's use snake case for key name.

```suggestion
      "set_temperature": "Select the temperature to set your module to"
```

---

this is taken from the field name which is why it is in camelcase